useful-scripts/get_data_block_responses/get_data_block_responses.py

```diff
@@ -104,22 +112,30 @@ def write_block_to_file(
         response_time,
         response_dict):
     try:
-        with open(f'{results_dir}/block_{block_guid}', 'w') as block_file:
-            block_file.write(
-                f'block: {block_guid}\n'
-                f'release: {release_guid}\n'
-                f'subject: {subject_guid}\n'
-                f'request:\n{json.dumps(request_dict, sort_keys=True, indent=2)}\n'
-                f'response status: {status_code}\n'
-                f'time for response: {response_time}\n'
-                f'response:\n{json.dumps(response_dict, sort_keys=True, indent=2)}'
-            )
+        with open(f'{responses_dir}/block_{block_guid}_response', 'w') as block_response_file:
+            with open(f'{requests_dir}/block_{block_guid}_request', 'w') as block_request_file:
```

Comment:
Rather than nest these, I'd have them separate
```
with open(....response) ...
    block_response_file.write(...)

with open(....request) ...
    block_request_file.write(...)
```

Reply:
Yup good point!  Updated, thanks